docs/manuals/auth/index.md

```diff
@@ -0,0 +1,53 @@
+# Taipy's Authentication and Authorization
+
+!!! warning "Reminder: Authorization and Authentication is only available in enterprise version"
+
+## Pre-requisite
+
+An LDAP server must be setup by the user. Taipy will not be responsible for setting the server up.
+
+## Config
+
+Set up connection to your LDAP server with Taipy Config. Here are the configurable properties you can use:
+
+- **Config.global_config.ldap_server**: The url of the ldap server.
+- **Config.global_config.ldap_base_dn**: The base DN that the LDAP server uses to search for users authentication.
+- **Config.global_config.jwt_secret**: The secret for the JWT token.
+- **Config.global_config.jwt_expire**: The period after which the JWT token provided by Taipy will be expired.
```

Comment:
this will result in changing the code in taipy-enterprise, I think it's best to leave it for now and have a different ticket for it. What do you think?

Reply:
I don't really mind. My point is that JWT is an implementation detail that users don't care to be exposed to.

Reply:
as discussed, we have removed this detail from the doc :D